docs/technical_reference/flowsheets/BSM2.rst

```diff
@@ -52,6 +61,132 @@ Documentation for each of the property models can be found here:
     * `ASM1 <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/ASM1.html>`_
```

Comment:
I think I replaced CSTR_with_injection with AerationTank, which inherits from CSTR with injection but sets certain config options.

Reply:
If so, we should list Aeration tank instead of CSTR w/injection